src/main/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionPriority.java

```diff
@@ -31,14 +31,14 @@ private KubernetesVersionPriority() {}
      * @return the version with the highest priority
      */
     public static String highestPriority(List<String> versions) {
-        List<Version> byPriority = getByPriority(versions);
+        List<KubernetesVersionFactory.Version> byPriority = getByPriority(versions);
```

Comment:
Here the import is not used anymore or if you want to use the import (which i prefer) it needs to be as before `List<Version>`

Reply:
good catch!